scm-webapp/src/main/java/sonia/scm/plugin/ExtensionCollector.java

```diff
@@ -42,20 +40,22 @@
 import java.util.Set;
 import java.util.stream.Collectors;
 
-//~--- JDK imports ------------------------------------------------------------
-
-/**
- *
- * @author Sebastian Sdorra
- */
-@SuppressWarnings("unchecked")
-public final class ExtensionCollector
-{
+@SuppressWarnings({"unchecked", "rawtypes", "java:S3740"})
```

Comment:
Just out of curiosity: Where is a violation of `java:S3740`?

Reply:
`java:S3740` is about raw type usage. There are a lot of `Class` generics in Lists, Sets, ...